### PR TITLE
Move to pyproject.toml

### PR DESCRIPTION
**Pull request by JulienPalard**
`setup.py` and `setup.cfg` are no longer needed, it's possible to build with `python -m build`.

### CODE REVIEW

**Comment by Seluj78** [comment]
Be careful to install and upgrade `build`. I've done the exact same thing on private repositories yesterday for work, and I had to do this :)

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -21,11 +21,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
+        pip install setuptools wheel twine build
```
Comment (JulienPalard) [inline .github/workflows/python-publish.yml:24]: Like this @Seluj78?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "potodo"
+description = "List .po files to be translated."
+authors = [
+    { name = "Jules Lasne", email = "jules.lasne@gmail.com" },
```
Comment (Seluj78) [inline pyproject.toml:9]: Do you want to add yourself @JulienPalard ?

**Review by Seluj78 — APPROVED** [review]
LGTM 👀😊

File: pyproject.toml (hunk 2 above)
Comment (JulienPalard) [reply pyproject.toml:9]: I may do, I may not, depends mainly on the wind direction ;)

**Pull request merged**